libs/coin-modules/coin-bitcoin/src/bridge/js.ts

```diff
@@ -88,7 +89,14 @@ function buildAccountBridge(signerContext: SignerContext, perfLogger: PerfLogger
 export type PerfLogger = {
   startSpan: StartSpan;
 };
-export function createBridges(signerContext: SignerContext, perfLogger: PerfLogger) {
+
+export function createBridges(
+  signerContext: SignerContext,
+  perfLogger: PerfLogger,
+  coinConfig: CoinConfig,
+) {
+  setCoinConfig(coinConfig);
```

Comment:
we need to add "ledger-live/libs/ledger-live-common/src/families/bitcoin/config.ts"
for the default config value for the deprecated coins. Right?

Reply:
Yeah the deprecated coins already have default values in config.ts

Reply:
https://github.com/LedgerHQ/ledger-live/blob/deprecated-coins-banner/libs/ledger-live-common/src/families/bitcoin/config.ts#L212

We let them as "active" for now and we'll change accordingly on Firebase when the time is right